README.md

```diff
@@ -10,10 +10,12 @@ This module consists of the following submodules:
 See more details in each module's README.
 
 ## Compatibility
-This module is meant for use with Terraform 0.13. If you haven't
-[upgraded](https://www.terraform.io/upgrade-guides/0-13.html) and need a Terraform
+This module is meant for use with Terraform 0.13+ and tested using Terraform 1.0+.
+If you find incompatibilities using Terraform `>=0.13`, please open an issue.
+
+If you haven't [upgraded][terraform-0.13-upgrade] and need a Terraform
 0.12.x-compatible version of this module, the last released version
-intended for Terraform 0.12.x is [v4.5.0](https://registry.terraform.io/modules/terraform-google-modules/-sql-db/google/v4.5.0).
+intended for Terraform 0.12.x is [12.3.0].
```

Comment:
`12.3.0` is not a valid SQL module release. `v5.0.0` is the last 0.12.x compat release.

Reply:
Done.